### PR TITLE
#8 contact page

### PR DESCRIPTION
**Pull request by samgildea**

### CODE REVIEW

**Comment by josephmannis** [comment]
Also seeing a gql error here: 
```
error There was an error in your GraphQL query:

Cannot query field "how_description" on type "PrismicProcessPageData".

If you don't expect "how_description" to exist on the type "PrismicProcessPageData" it is most likely a typo.
However, if you expect "how_description" to exist there are a couple of solutions to common problems:

- If you added a new data source and/or changed something inside gatsby-node.js/gatsby-config.js, please try a restart of your development server
- The field might be accessible in another subfield, please try your query in GraphiQL and use the GraphiQL explorer to see which fields you can query and what shape they have
- You want to optionally use your field "how_description" and right now it is not used anywhere. Therefore Gatsby can't infer the type and add it to the GraphQL schema. A quick fix is to add at least one entry with that field ("dummy content")

It is recommended to explicitly type your GraphQL schema if you want to use optional fields. This way you don't have to add the mentioned "dummy content". Visit our docs to learn how you can define the schema for "PrismicProcessPageData":
https://www.gatsbyjs.org/docs/schema-customization/#creating-type-definitions
```
I can review if you can resolve this real quick. Thanks!

**Comment by samgildea** [comment]
Those graphql errors should be fixed for now.  A couple of the schemas have been updated for the process page and the solutions page which is causing them.

File: src/components/contact/contact.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    <ContactContainer>
+      <ContactSection>
+        <ContactForm>
+          <PlantIcon />
+          <H2>{data.prismicContactPage.data.overview_heading}</H2>
+          <ContactDescription>
+            <Body>{data.prismicContactPage.data.overview_description}</Body>
+          </ContactDescription>
+          <form>
+            <label>
```
Comment (josephmannis) [inline src/components/contact/contact.js]: Need form labels here:
https://www.w3.org/WAI/tutorials/forms/

Also the typography does not match the hi-fi designs.

**Review by josephmannis — CHANGES_REQUESTED** [review]
Just two things!

**Pull request merged**